lib/cell.rb

```diff
@@ -0,0 +1,27 @@
+class Cell
+  attr_reader :coordinate, :ship
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    #When place_ship method is called, call.ship will contian ship.
+    @ship = nil
+  end
+
+  def empty?
+    @ship.nil?
+  end
+
+  def place_ship(ship)
+    @ship = ship
```

Comment:
I did everything the same as you up until here, where I wondered if we should use a hash for cells and ships? That might be something we introduce later, though.